tests/test_convert.py

```diff
@@ -799,6 +799,24 @@ def test_convert_cytominerdatabase_csv(
                 source_datatype="csv",
                 join=True,
                 drop_null=False,
+                # cytominer-database test datasets don't include image FileName columns
```

Comment:
because cytominer-database is deprecated, it might not be wise to add new references to it. This type of legacy documentation will become irrelevant, IMO.

Reply:
Thanks! Made an update to this to make this more of a generic reference.